lib/elastic_record/doctype.rb

```diff
@@ -10,18 +10,6 @@ class Doctype
       }
     }
 
-    PERCOLATOR_MAPPING = {
-      properties: {
-        query: {
-          type: "percolator"
-        }
-      }
-    }
-
-    def self.percolator_doctype
-      new('queries', PERCOLATOR_MAPPING)
-    end
```

Comment:
Multi-doctype no longer allowed. Percolators should now share the same doctype with an additional `query: {type: percolator}` field added to the mapping, or (more ideally), be contained in their own separate index.